Userland/Libraries/LibPDF/Renderer.cpp

```diff
@@ -9,6 +9,16 @@
 #include <ctype.h>
 #include <math.h>
 
+#define RENDERER_HANDLER(name) \
+    void Renderer::handle_##name([[maybe_unused]] const Vector<Value>& args)
```

Comment:
\<rant> I don't know how I feel about having more of these, the javascript one is sorta acceptable because it has a bunch of parameters that are too annoying to write and it has the benefit of clearly marking functions as exported...
Not sure if this is adding any such value here beyond convenience \</rant>

(feel free to ignore this, it's more of a thought dump than an actual complaint)

Reply:
What's wrong with convenience? (not trying to be snarky or anything, genuinely curious as I'm fairly new to the C++ world)

Reply:
It's a great thing, but at some point the code becomes an unreadable mess of macros.
This change is fine, I was in a pretty shit mood when I wrote that lol